test/functional/wallet-hd.py

```diff
@@ -10,6 +10,7 @@
     connect_nodes_bi,
 )
 import shutil
+import os
```

Comment:
nit: please sort imports :)

and ideally place them in PEP8 ordering (ie standard library before project imports)